src/index.ts

```diff
@@ -21,20 +24,46 @@ export const createKZG = async () => {
     const verifyBlobKzgProofWasm = module.cwrap('verify_blob_kzg_proof_wasm', 'string', ['array', 'array', 'array']) as (blob: Uint8Array, commitment: Uint8Array, proof: Uint8Array) => string
     const verifyKzgProofWasm = module.cwrap('verify_kzg_proof_wasm', 'string', ['array', 'array', 'array', 'array']) 
 
-    const loadTrustedSetup = (trustedSetup: TrustedSetup) => {
+    /**
+     * 
+     * @param trustedSetup - an optional trusted setup parameter provided by the user
+     * @returns 0 if loaded successfully or 1 otherwise
+     */
+    const loadTrustedSetup = (trustedSetup?: TrustedSetup) => {
```

Comment:
```suggestion
    const loadTrustedSetup = (trustedSetup: TrustedSetup = mainnetTrustedSetup) => {
```
small nit, and then we can remove the undefined check